CRM/Chreports/Reports/ReportConfiguration.php

```diff
@@ -355,15 +354,18 @@ public function buildJsonConfigSettings(): void {
                 if($vOrder['column'] == '-') {
                     continue;
                 }
-                // Check for custom field
-                // if it's a custom field, get the key value from mapping.json
-                $orderByKey = $vOrder['column'];
-                if(strpos($orderByKey, 'custom_') === 0) {
-                    $orderByKey = $this->getCustomFieldFromMappingJson($orderByKey);
-                    unset($vOrder['column']);
+                // check whether the field is part of from mapping.json
+                $orderByField = $this->getFieldFromMappingJson($vOrder['column']);
+                if(empty($orderByField)) {
+                    $config['log_messages'][] = 'Unable to port OrderBy Field "'. $vOrder['column'] . '", no mapping found';
```

Comment:
Adding regular log for easy access.
```suggestion
                    watchdog("reporting", "Failed to map field: ".$vOrder['column'] . " -> ".$config['title'] . "", [], WATCHDOG_DEBUG);
                    $config['log_messages'][] = 'Unable to port OrderBy Field "'. $vOrder['column'] . '", no mapping found';
```